simulation/utils/simulation_pygame_utils.py

```diff
@@ -1,16 +1,42 @@
 import pygame
+import pymunk
+
 from pygame.color import THECOLORS
 
-from utils.simulation_pymunk_utils import SCREEN_HEIGHT
+from utils.simulation_pymunk_utils import Color
+
+GOAL_OBJECT_COLOR = BLUE = Color(51, 153, 255)
```

Comment:
Lepiej. lepiej :+1:  Aczkolwiek nadal sprawdziłbym czy nie ma podefiniowanych w bibliotece jakiś stałych na kolory

Reply:
Niżej widzę jakieś THECOLORS["red"], możliwe, że właśnie o to mi chodzi

Reply:
yup, dziala analogia dla innych kolorów